pyproject.toml

```diff
@@ -26,6 +26,12 @@ dependencies = [
     "requests",
     "h3ronpy>=0.18.0",
     "osmnx",
+    "omegaconf",
+    "python-dotenv",
+    "datasets",
+    "contextily",
+    "folium",
+    "mapclassify"
```

Comment:
Leave only omegaconf and datasets here. Dotenv is not needed and last 3 are plotting related so we have it already included in extra "plotting"